src/snowcli/docs/templates/usage.rst.jinja2

```diff
@@ -0,0 +1,23 @@
+{{ help }}
+
+Syntax
+===============================================================================
+
+.. code-block:: console
```

Comment:
I wonder if we can describe somewhere how we can preview generated docs. Intellij's preview doesn't look good:

![Screenshot 2023-08-01 at 00 04 02](https://github.com/Snowflake-Labs/snowcli/assets/132934591/b17f5831-a05f-455a-b2d1-5f5a35c804ce)

Reply:
Personally I use raw rst. I think proper preview will be part of next steps when we connect with the rest of the docs